src/App.tsx

```diff
@@ -5,7 +5,13 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovie] = useState<Movie[]>([]);
+
+  const handleAddMovie = (movie: Movie) => {
+    if (!movies.some(newMovie => newMovie.imdbId === movie.imdbId)) {
```

Comment:
newMovie not the best name for this variable, because you take an item from existing array.

Reply:
Agree with @LesyaSvk, rename it to smth like `existingMovie`